main/demo_tasks/ota_over_mqtt_demo/ota_over_mqtt_demo.c

```diff
@@ -898,7 +897,7 @@ static uint16_t getFreeOTABuffers( void )
 
     if( xSemaphoreTake( bufferSemaphore, portMAX_DELAY ) == pdTRUE )
     {
-        for( ulIndex = 0; ulIndex < MAX_NUM_OF_OTA_DATA_BUFFERS; ulIndex++ )
+        for( ulIndex = 0; ulIndex < otaconfigMAX_NUM_OTA_DATA_BUFFERS; ulIndex++ )
```

Comment:
Suggest we use the name `otademoconfigMAX_NUM_OTA_DATA_BUFFERS`
```suggestion
        for( ulIndex = 0; ulIndex < otademoconfigMAX_NUM_OTA_DATA_BUFFERS; ulIndex++ )
```

Reply:
Updated